.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,11 +14,15 @@ Thanks for contributing! Please remove any top-level sections that do not apply
 
 Description of the changes here.
 
+## Issues fixed
+
+Link any issues this PR closes here.
```

Comment:
Should we do this, or make it a checkbox asking to link them in the right sidebar?